canonical/ubuntu.com
# Fix for tablet

## Done
- Added some styling to fix `for-medium`
- Updated `for-tablet` to `for-medium`
## QA
- Go to /management/ubuntu-advantage
- Check the hero image only appears once
- Check different screen sizes
## Issue / Card
- FIxes https://github.com/ubuntudesign/www.ubuntu.com/issues/241
